packages/components/src/components/checkbox/_checkbox.scss

```diff
@@ -46,7 +50,6 @@
   // creating our own checkbox style, we'll need to position this in order to
   // accomodate the spacing needed for the checkbox.
   .#{$prefix}--checkbox-label {
-    @include reset;
```

Comment:
Removing this because I've moved it to the parent component, the wrapper, added to line 19.